### ISSUE TITLE
minikube docker-env writes error to stdout if VM isn't running - confuses shell

### ISSUE THREAD

**Issue by misha-tectonic** [kubernetes/minikube#3311]
**Environment**:

**Minikube version**: 0.30
- **OS** (e.g. from /etc/os-release): macOS 10.13
- **VM Driver**: VirtualBox
- **ISO version**: N/A
- **Install tools**:
- **Others**:

**What happened**:

```
% eval $(minikube docker-env)
E1107 17:14:21.381360    3836 env.go:330] Error setting machine env variable(s): Error getting ip from host: Host is not running
zsh: =============================================================================== not found
```

**What you expected to happen**:

If a Minikube VM is not running, `eval $(minikube docker-env)` should fail, but not in such a confusing way, with shell trying to execute a command `===============================================================================`.

**How to reproduce it** (as minimally and precisely as possible):

Stop the Minikube VM. Run `eval $(minikube docker-env)`

**Comment by misha-tectonic** [comment]
```
% minikube status
minikube: Stopped
cluster:
kubectl:
```

```
% minikube docker-env
E1108 17:10:20.272974   76458 env.go:330] Error setting machine env variable(s): Error getting ip from host: Host is not running
================================================================================
An error has occurred. Would you like to opt in to sending anonymized crash
information to minikube to help prevent future errors?
To opt out of these messages, run the command:
	minikube config set WantReportErrorPrompt false
================================================================================
Please enter your response [Y/n]: n
```

**Comment by misha-tectonic** [comment]
And indeed the crash reporter prints to stdout.

**Comment by tstromberg** [comment]
That sounds reasonable. I'd be happy to review any PR's or help anyone interested in addressing this bug.

**Comment by tstromberg** [comment]
Fixed in v0.34.0. You may want to be aware of #3697 though.
